back-end/api/quickstart/models.py

```diff
@@ -24,12 +24,12 @@ class Visibility(models.TextChoices):
     PUBLIC = 'Public'
     FRIENDS = 'Friends'
 
-  id = models.CharField(primary_key=True, max_length=LONG_CHAR_LENGTH)
+  id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
   type = 'post'
-  title = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  title = models.TextField()
   description = models.TextField()
-  source = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
-  origin = models.CharField(max_length=LONG_CHAR_LENGTH, unique=True)
+  source = models.TextField()
+  origin = models.TextField()
   visibility = models.CharField(max_length=SHORT_CHAR_LENGTH, choices=Visibility.choices)
   unlisted = models.BooleanField()
   author = models.CharField(max_length=LONG_CHAR_LENGTH)
```

Comment:
should these other char fields be changed to textfields as well?

Reply:
Gonna leave this for a future PR for someone to change all of them

Didn't wanna pollute this PR (I changed these ones on this PR since they shouldn't have been `unique=True`)